docs/guides/migrations.md

```diff
@@ -7,6 +7,15 @@ kb_sync_latest_only
 
 # Migrations
 
+## 4.1 to 4.2
+
+A better handling for cookie `SameSite` and `secure` settings was implemented with new defaults to `SameSite=Strict` and `secure`.
+This can still be overridden when calling `cookies.services` `put` method with explicitly set values.
+Additionally if the PWA is run in an iframe all cookies are set with `SameSite=None` and if run with `http` (should only be in development environments) `secure` is not set.
+For migrating the occurrence of `cookies.services` `put` can be check whether they need to be adapted.
```

Comment:
For migrating check the calls of `cookies.services` `put` method whether they need to be adapted. (maybe this should be the last sentence of this block)